config/routes.rb

```diff
@@ -3,9 +3,16 @@
   resources :merchants do
 
     resources :invoices, controller: 'merchant_invoices'
-    
+
     resources :items, controller: 'merchant_items', except: [:destroy]
 
     resources :dashboard, controller: 'merchant_dashboard', only: [:index]
   end
+
+  resources :admin, only: [:index]
+
+  scope :admin do
```

Comment:
will this create a merge conflict with @gregoryjflaherty 's #66 PR for us21 since that has namespace? Is there a benefit to one vs the other?

Reply:
It will, evidently scope wasn't working for greg so he changed it to namespace. I am fine with namespace if it is what works.

Reply:
Yeah scope wont work once you start having to route to other controllers - there is a way - but the way we have it set up wont work.